packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -68,53 +67,25 @@ const streamMessagesApi = orchestratorApi.injectEndpoints({
                     updateCachedData(() => false);
                 };
 
-                const invalidateTag = (cacheTag: CacheTags, id?: string) => {
-                    if (validCacheTags.includes(cacheTag)) {
-                        // If we receive an object with cacheTag and id we both invalidate the cache key and the cache key with the id
-                        // invalidating both the general and specific caches
-                        const cacheTags: CacheInvalidationTag[] = [cacheTag];
-
-                        if (id) {
-                            cacheTags.push({
-                                type: cacheTag,
-                                id,
-                            });
+                const invalidateTag = (cacheTag: CacheTag) => {
+                    const tagType = (() => {
+                        if (typeof cacheTag === 'object') {
```

Comment:
According to the type definition in types.ts it is always an object, therefore this check is redundant.

Reply:
fixed